core/admin.py

```diff
@@ -1,30 +1,46 @@
 from django.contrib import admin
 from django.forms import ModelForm
 from django.contrib.auth import admin as auth_admin
+from django.utils import timezone
+from django_date_extensions.fields import ApproximateDate
 from suit_redactor.widgets import RedactorWidget
-from suit.admin import SortableModelAdmin
+from suit.admin import SortableModelAdmin, SortableTabularInline
 
 from .models import *
 from .forms import UserChangeForm, UserCreationForm, UserLimitedChangeForm
 
+now = timezone.now()
```

Comment:
I think this will be a problem because it will be set to the date when the server is started (and the file is read).